src/F2_autoreport.py

```diff
@@ -0,0 +1,119 @@
+import dash
+import dash_html_components as html
+import time
+
+app = dash.Dash(__name__)
+
+colors = {
+    'bg-head': '#9ae6db',
+    'text-head': '#000000',
+    'text-body': '#000000',
+    'inp-box': '#03034f',
+    'font-inpbox': '#FFFFFF'
+}
+
+projectName = 'Harbor Norway'
+scenarioName = '100% self-generation'
+releaseDesign = '0.0x'
+branchID = 'xcdd5eg004'
+simDate = time.strftime("%Y-%m-%d")
+
+# Header section with logo and the title of the report, and CSS styling. Work in progress...
+
+app.layout = html.Div([
+
+    html.Div(className='header_title_logo',
+             children=[html.Img(id='mvslogo', src='https://elandh2020.eu/wp-content/themes/netron/libs/images/logo'
+                                                  '/logo-eland-original.svg', width='370px'),
+                       html.H1('MULTI VECTOR SIMULATION - REPORT SHEET')],
+             style={
+                 'textAlign': 'center',
+                 'color': colors['text-head'],
+                 'borderStyle': 'solid',
+                 'borderWidth': 'thin',
+                 'padding': '10px',
+                 'margin': '30px',
+                 'fontSize': '225%'
+             }),
+
+    html.Div(className='imp-info', children=[
+        html.P(f"MVS Release: {releaseDesign}"),
+        html.P(f"Branch-id: {branchID}"),
+        html.P(f"Simulation date: {simDate}")
+
+    ],
+             style={
+                 'textAlign': 'right',
+                 'padding': '5px',
+                 'fontSize': '22px',
+                 'margin': '30px'
+
+             }),
+
+    html.Div(className='imp_info2', children=[
+        html.Div([html.Span('Project name   : ', style={'fontWeight': 'bold'}), f'{projectName}']),
+        html.P(''),
```

Comment:
Why is this empty P needed?

Reply:
Used to create a white space. I will replace it with html.Br() which is the right way

Reply:
Using css to change margin or add padding could also be a solution :)